### ISSUE TITLE
[Anon] Bug report: Blood Bowl.gstz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/bloodbowl#83]
**File:** Blood Bowl.gstz

**BattleScribe version:** 2.03.27

**Platform:** Android

**Dropbox:** No

**Description:** Increasing "Seasons played" does not increase players TV 20k

**Comment by krobb001** [comment]
I assume this refers to the "Agent fee" on page 100 of the rulebook, where the cost of re-drafting a player that has played in one or more seasons is increased by 20k for each prior season played. This does not, at least to my understanding, contribute to the player's Team Value, but instead only increases the GP fee to re-draft said player

@Gasparen I'm pretty sure this is correct, but if you could double check this before closing 🙏

**Comment by Gasparen** [comment]
Yeah, that is correct. We currently only show the CTV of each team and not the cost, so we have no way of showing this cost increase
